src/java/org/apache/cassandra/tools/BulkLoader.java

```diff
@@ -67,9 +73,11 @@ public static void load(LoaderOptions options) throws BulkLoadException
                         handler,
                         options.connectionsPerHost,
                         options.targetKeyspace);
-        DatabaseDescriptor.setStreamThroughputOutboundMegabitsPerSec(options.throttle);
-        DatabaseDescriptor.setInterDCStreamThroughputOutboundMegabitsPerSec(options.interDcThrottle);
-        StreamResultFuture future = null;
+        DatabaseDescriptor.setStreamThroughputOutboundMegabitsPerSec(Double.valueOf(options.throttleMegabits).intValue());
+        DatabaseDescriptor.setInterDCStreamThroughputOutboundMegabitsPerSec(Double.valueOf(options.interDcThrottleMegabits).intValue());
```

Comment:
I guess it is not a big deal but I wanted to mention again that if we deprecate the megabits options and we go for mebibytes internally in DataRateSpec, probably better to add a setter in mebibytes and use mebibytes and do only one calculation megabits to mebibytes in the parking area for legacy reason in case someone wants to use the old ones.

Reply:
I've added two setters `setStreamThroughputOutboundMebibytesPerSec`, `setInterDCStreamThroughputOutboundMebibytesPerSec`. However, I've decided to keep `throttle` and `interDcThrottle` in megabits, because the conversion from mebibytes to megabits produces a larger number, and it is less prone to loss of precision.

For example, if we have 0.1 and 0.3 mebibytes, the rounded conversion to megabits will be 1, and 3 megabits respectively. Because we cast these numbers to `int` in the setter, we get more precision in megabits. If we go the other way around, we lose precision, 3 and 1 both end up as 0 mebibytes.

Reply:
Considering you kept throttleMegabits and interDcThrottleMegabits being double, I guess this comment is outdated. 
I spent some time thinking and I think we should just make internally those 4 properties to be operated in bytes and change also throttleMegabits and interDcThrottleMegabits to being bytes. This will save a lot of calculations and headache. 
If we step back and look at the StreaManager.StreamRateLimiter, we will see that the rate limiter expects bytes.
Why this was actually not done when we were switching to the new types - because the whole effort was huge and switching internally parameters to smallest possible unit was left for follow up improvement tickets. But in this case I think this is at this point more like a bug considering how many twists and calculations the bulk loader will trigger. WDYT? CC @yifan-c